src/praisonai/praisonai/cli/features/doctor/checks/config_checks.py

```
@@ -168,8 +168,14 @@ def check_agents_yaml_schema(config: DoctorConfig) -> CheckResult:
     # Check for common fields
     if "framework" in data:
         framework = data["framework"]
-        if framework not in ["praisonai", "crewai", "autogen"]:
-            warnings.append(f"Unknown framework: {framework}")
+        try:
+            from praisonai.framework_adapters.registry import get_default_registry
+            registered = get_default_registry().list_names()
+            if framework not in registered:
+                warnings.append(f"Unknown framework: {framework} (registered: {', '.join(sorted(registered))})")
+        except Exception:
```

Ruff (0.15.18)

[warning] Do not catch blind exception: `Exception`

(BLE001)